Error resolving status.mojang.com

This appears to be caused by Mojang shutting down the status service.
https://wiki.vg/Mojang_API#API_Status_-_Outdated

I will try and write a PR that removes the status API.

Full stack trace:
```
java.lang.RuntimeException: com.mashape.unirest.http.exceptions.UnirestException: java.net.UnknownHostException: status.mojang.com: No address associated with hostname
        at org.shanerx.mojang.Mojang.getJSONArray(Mojang.java:406) ~[survivalserver-1.3.1.jar:?]
        at org.shanerx.mojang.Mojang.connect(Mojang.java:70) ~[survivalserver-1.3.1.jar:?]
        at net.introvertscove.survivalserver.plugin.utils.UUIDFetcher.init(UUIDFetcher.java:21) ~[survivalserver-1.3.1.jar:?]
        at net.introvertscove.survivalserver.plugin.utils.UUIDFetcher.getLastName(UUIDFetcher.java:164) ~[survivalserver-1.3.1.jar:?]
        at net.introvertscove.survivalserver.plugin.utils.UUIDFetcher$3.run(UUIDFetcher.java:127) ~[survivalserver-1.3.1.jar:?]
        at org.bukkit.craftbukkit.v1_17_R1.scheduler.CraftTask.run(CraftTask.java:101) ~[patched_1.17.1.jar:git-Paper-327]
        at org.bukkit.craftbukkit.v1_17_R1.scheduler.CraftAsyncTask.run(CraftAsyncTask.java:57) ~[patched_1.17.1.jar:git-Paper-327]
        at com.destroystokyo.paper.ServerSchedulerReportingWrapper.run(ServerSchedulerReportingWrapper.java:22) ~[patched_1.17.1.jar:git-Paper-327]
        at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1130) ~[?:?]
        at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:630) ~[?:?]
        at java.lang.Thread.run(Thread.java:831) ~[?:?]
Caused by: com.mashape.unirest.http.exceptions.UnirestException: java.net.UnknownHostException: status.mojang.com: No address associated with hostname
        at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:143) ~[survivalserver-1.3.1.jar:?]
        at com.mashape.unirest.request.BaseRequest.asString(BaseRequest.java:56) ~[survivalserver-1.3.1.jar:?]
        at org.shanerx.mojang.Mojang.getJSONArray(Mojang.java:401) ~[survivalserver-1.3.1.jar:?]
        ... 10 more
Caused by: java.net.UnknownHostException: status.mojang.com: No address associated with hostname
        at java.net.Inet6AddressImpl.lookupAllHostAddr(Native Method) ~[?:?]
        at java.net.InetAddress$PlatformNameService.lookupAllHostAddr(InetAddress.java:932) ~[?:?]
        at java.net.InetAddress.getAddressesFromNameService(InetAddress.java:1517) ~[?:?]
        at java.net.InetAddress$NameServiceAddresses.get(InetAddress.java:851) ~[?:?]
        at java.net.InetAddress.getAllByName0(InetAddress.java:1507) ~[?:?]
        at java.net.InetAddress.getAllByName(InetAddress.java:1366) ~[?:?]
        at java.net.InetAddress.getAllByName(InetAddress.java:1300) ~[?:?]
        at org.apache.http.impl.conn.SystemDefaultDnsResolver.resolve(SystemDefaultDnsResolver.java:45) ~[survivalserver-1.3.1.jar:?]
        at org.apache.http.impl.conn.DefaultHttpClientConnectionOperator.connect(DefaultHttpClientConnectionOperator.java:111) ~[survivalserver-1.3.1.jar:?]
        at org.apache.http.impl.conn.PoolingHttpClientConnectionManager.connect(PoolingHttpClientConnectionManager.java:353) ~[survivalserver-1.3.1.jar:?]
        at org.apache.http.impl.execchain.MainClientExec.establishRoute(MainClientExec.java:380) ~[survivalserver-1.3.1.jar:?]
        at org.apache.http.impl.execchain.MainClientExec.execute(MainClientExec.java:236) ~[survivalserver-1.3.1.jar:?]
        at org.apache.http.impl.execchain.ProtocolExec.execute(ProtocolExec.java:184) ~[survivalserver-1.3.1.jar:?]
        at org.apache.http.impl.execchain.RetryExec.execute(RetryExec.java:88) ~[survivalserver-1.3.1.jar:?]
        at org.apache.http.impl.execchain.RedirectExec.execute(RedirectExec.java:110) ~[survivalserver-1.3.1.jar:?]
        at org.apache.http.impl.client.InternalHttpClient.doExecute(InternalHttpClient.java:184) ~[survivalserver-1.3.1.jar:?]
        at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:82) ~[survivalserver-1.3.1.jar:?]
        at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:107) ~[survivalserver-1.3.1.jar:?]
        at org.apache.http.impl.client.CloseableHttpClient.execute(CloseableHttpClient.java:55) ~[survivalserver-1.3.1.jar:?]
        at com.mashape.unirest.http.HttpClientHelper.request(HttpClientHelper.java:138) ~[survivalserver-1.3.1.jar:?]
        at com.mashape.unirest.request.BaseRequest.asString(BaseRequest.java:56) ~[survivalserver-1.3.1.jar:?]
        at org.shanerx.mojang.Mojang.getJSONArray(Mojang.java:401) ~[survivalserver-1.3.1.jar:?]
        ... 10 more
```

--- a/README.md
+++ b/README.md
@@ -54,14 +54,7 @@ dependencies {
 Once you have imported the library, use the following to establish a connection with Mojang's servers.
 ```java
 Mojang api = new Mojang().connect();
-```
-You can ping the api like this:
-```java
-if (api.getStatus(Mojang.ServiceType.AUTH_MOJANG_COM) != Mojang.ServiceStatus.GREEN) {
-	System.err.println("The Auth Server is not available right now.");
-}
-```
-to check whether authentication servers are currently down.
+```      
 
 ## Documentation
 * Generated HTML-Javadoc files are available for download in HTML-format in the `docs/` directory.

--- a/src/main/java/org/shanerx/mojang/Mojang.java
+++ b/src/main/java/org/shanerx/mojang/Mojang.java
@@ -44,17 +44,12 @@ import com.mashape.unirest.http.exceptions.UnirestException;
  */
 @SuppressWarnings({ "unchecked" })
 public class Mojang
-{
-
-    private Map<String, ServiceStatus> apiStatus;
+{    
 
     /**
      * Constructor. Initializes member variables.
      */
-    public Mojang()
-    {
-        apiStatus = new HashMap<>();
-    }
+    public Mojang() {}
 
     /**
      * <p>Opens the connection with the Mojang API.
@@ -67,35 +62,18 @@ public class Mojang
      */
     public Mojang connect()
     {
-        JSONArray arr = getJSONArray("https://status.mojang.com/check");
-
-        for (int i = 0; i <= 7; ++i) {
-            ServiceType st = ServiceType.values()[i];
-            String service = st.toString();
-            JSONObject obj = (JSONObject) arr.get(i);
-
-            apiStatus.put(service,
-                    ServiceStatus.valueOf(((String) obj.get(service)).toUpperCase()));
-        }
-
         return this;
     }
 
     /**
-     * Retrieves the {@link org.shanerx.mojang.Mojang.ServiceStatus status} of a portion of the API.
-     * Check the enum entries for {@link org.shanerx.mojang.Mojang.ServiceStatus} for possible response types.
-     *
-     * @param service the service type
-     *
-     * @return the status of said service
+     * @deprecated This service was closed down by Mojang on October 8th 2021 due to incorrect status responses.
+     * https://wiki.vg/Mojang_API#API_Status_-_Outdated
+     *  
+     * https://bugs.mojang.com/browse/WEB-2303?focusedCommentId=1086543&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-1086543
      */
     public ServiceStatus getStatus(ServiceType service)
-    {
-        if (service == null)
-        {
-            return ServiceStatus.UNKNOWN;
-        }
-        return apiStatus.get(service.toString());
+    {       
+    	return ServiceStatus.UNKNOWN;        
     }
 
     /**

